test/test_vending_machine.py

```diff
@@ -44,17 +49,19 @@ def test_buy_product_with_payment_expects_product(self):
         result = self.vending_machine.buy_product()
 
         # Assert
+        assert_is_not_none(result)
         assert_that(result, is_(not_none()))
 
-    @raises(RuntimeError)
+    #@raises(RuntimeError)
```

Comment:
why are we commenting out `@raises`?  We specifically call this out in our slides (unless you changed those too?)

More to the point, is this a better practice?  With `assert_raises` I don't think we can run multiple lines of code unless you create a lambda function.  In other languages we just use a &block or closure.